test/test_images/emptydir/emptydir.go

```diff
@@ -18,37 +18,36 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"net/http"
 	"os"
+	"path"
 
 	"knative.dev/serving/test"
 )
 
-var path string
+func testfilePath() string {
+	base := os.Getenv("DATA_PATH")
+	if base == "" {
+		base = "/data"
+	}
+	return path.Join(base, "testfile")
```

Comment:
super-nit, but should probably be filepath.Join

Reply:
Done.